CMakeLists.txt

```diff
@@ -45,4 +45,4 @@ add_library(json INTERFACE)
 target_include_directories(json INTERFACE ${json_SOURCE_DIR}/include)
 
 # link all libs
-target_link_libraries(engine_computer PRIVATE ${LJM} ${BOOST_Libraries} json)
+target_link_libraries(engine_computer PRIVATE pthread ${LJM} ${BOOST_Libraries} json)
```

Comment:
what is pthread linked for?

Reply:
uhh the linking fails for me on WSL without it, is there something I'm missing? maybe updating ubuntu to the latest version?

Reply:
oh i guess that's fine then, it's a gcc version moment probably

Reply:
probably 💀 I should probably just update wsl ubuntu instead of using the LTS version from 20 years ago